help/destinations/catalog/ecommerce/sap-commerce.md

```diff
@@ -0,0 +1,329 @@
+| --- | --- | --- |
+| `xdm: b2b.companyName` | `Attribute: company` | Yes |
+| `xdm: workAddress.countryCode` | `Attribute: countryCode` | Yes |
+
+>[!ENDTABS]
+
+#### Mapping additional attributes
+
+You can then add any additional mappings between your XDM profile schema and the [!DNL SAP Subscription Billing] [schema](https://api.sap.com/api/BusinessPartner_APIs/schema) attributes for a customer as shown below:
+
+>[!BEGINTABS]
+
+>[!TAB Individual customer]
+
+| Source Field | Target Field | Mandatory |
+| --- | --- | --- |
+| `xdm: person.name.firstName` | `Attribute: firstName` |
+| `xdm: workAddress.street1` | `Attribute: street` |
+| `xdm: workAddress.city` | `Attribute: city` |
+
+An example with both mandatory and optional attribute mappings where the customer is an individual is shown below:
+![Image from Platform UI showing an example with both mandatory and optional attribute mappings where the customer is an individual.](../../assets/catalog/ecommerce/sap-commerce/mapping-attributes-individual.png)
+
+>[!TAB Corporate customer]
+
+| Source Field | Target Field | Mandatory |
+| --- | --- | --- |
+| `xdm: person.name.firstName` | `Attribute: firstName` |
+| `xdm: workAddress.street1` | `Attribute: street` |
+| `xdm: workAddress.city` | `Attribute: city` |
+
+An example with both mandatory and optional attribute mappings where the customer is a corporate is shown below:
+![Image from Platform UI showing an example with both mandatory and optional attribute mappings where the customer is a corporate.](../../assets/catalog/ecommerce/sap-commerce/mapping-attributes-corporate.png)
+
+>[!ENDTABS]
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+### Schedule audience export and example {#schedule-segment-export-example}
+
+When performing the [Schedule audience export](/help/destinations/ui/activate-segment-streaming-destinations.md#scheduling) step, you must manually map Platform audiences to the [attributes](#prerequisites-attribute) in [!DNL SAP Commerce].
+
+To do this, select each segment, then enter name of the custom reference from [!DNL SAP Subscription Billing] in the [!DNL SAP Commerce] **[!UICONTROL Mapping ID]** destination connector field. For guidance on creating custom references, refer to the [Create custom references in [!DNL SAP Subscription Billing]](#prerequisites-custom-reference) section.
+
+>[!IMPORTANT]
+>
+> Do not use the custom reference label as the value.
+![Image indicating you should not use the custom reference label value for mapping.](../../assets/catalog/ecommerce/sap-commerce/custom-reference-dont-use-label-for-mapping.png)
+
+For example, if your [!DNL SAP Subscription Billing] custom reference is `SAP_1`, specify this value in the [!DNL SAP_Commerce] **[!UICONTROL Mapping ID]** to populate audience audiences from Experience Platform into this attribute.
+
+An example attribute from [!DNL SAP Subscription Billing] is shown below:
```

Comment:
I don't really understand this screenshot. can we detail please what 1 and 2 in the screenshot mean?
I also see that SAP_1 is not highlighted in the SAP UI, but in the next screenshot I see that is the value that we need to use.

Reply:
Was attempting to convey the attribute creation steps. Anyways removed the numbering and image has been updated to align with the subsequent image.